tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java

```diff
@@ -19,14 +19,651 @@
 import com.ibm.icu.util.Output;
 
 public class PathDescription {
+    /**
+     * Remember to quote any [ character!
+     */
+    final private static String pathDescriptionString = ""
```

Comment:
I'm surprised by this,  why is this better than `PathDescription.txt` ?

Reply:
For encapsulation and maintainability. This way we can use the CLDRURL constants instead of the string literals, and we can use them for path descriptions the same way we use them in other java files.

PathDescription.txt contains the same literal URLs repeatedly, embedded inside long message strings, which in turn are concatenated to regex followed by semi-colons. Those long lines are a pain in the horizontal scrollbar.

I considered implementing a substitution mechanism so the CLDRURL constants could be used in the .txt file. Then I realized what a waste of time that would be, and also how inferior. For example, the java compiler will tell us if we use a constant that's missing from CLDRURL. We don't need to invent a new file format with its own macro language. Then I noticed that we already had loadFromString :-)